gluster_brick_provisioning.adoc

```diff
@@ -0,0 +1,41 @@
+// vim: tw=79
+= Gluster brick provisioning
+
+This document describes some of the conventions followed when tendrl provisions
+bricks for gluster volumes. Also it provides guidelines that user has to follow
+if he creates brick manually and tendrl has to pick that.
+
+== Conventions followed by tendrl for Gluster brick creation
+
+ * User will have to provide just the brick name, disk and node id.
+
+ * Mount point for the brick will be created at "/tendrl_gluster_bricks/<brick_name>_mount"
+
+ * The actual brick path will be "/tendrl_gluster_bricks/<brick_name>_mount/<brick_name>"
+
+ * The brick will made up of lvm logical volume and naming convention used for
+   lvm entities are as follows:
+      - pv   : tendrl_<brick path with "/" replaced by "_">_pv
+      - vg   : tendrl_<brick path with "/" replaced by "_">_vg
+      - pool : tendrl_<brick path with "/" replaced by "_">_pool
+      - lv   : tendrl_<brick path with "/" replaced by "_">_lv
+
+ * Example for a brick with path "/tendrl_gluster_bricks/brick1_mount/brick1":
```

Comment:
What if I have 2 bricks on paths "/tendrl_gluster_bricks/brick1_mount/brick1" and "/tendrl_gluster_bricks/brick1/mount_brick1"? Would it break something? If so, would it make sense to add some hash to the generated pv/vg name?